rpcs3/Emu/Cell/Modules/cellAdec.cpp

```diff
@@ -467,7 +467,7 @@ class AudioDecoder : public ppu_thread
 						else
 						{
 							data = NULL;
-							size = 0;
+							this->size = NULL;
```

Comment:
Why are you assigning a pointer to an int?
```suggestion
							this->size = 0;
```

Reply:
Yes, you're right.

Reply:
Fix it.
https://github.com/RPCS3/rpcs3/pull/9346/commits/e1a7573c581f66e3d529f9aa743dc759ad26b179